docs/src/contribute/design/mqtt-topic-design.md

```diff
@@ -0,0 +1,177 @@
+---
+title: MQTT Topic Design
+tags: [Documentation]
+sidebar_position: 2
+draft: true
+---
+
+# MQTT Topic Design
+
+A detailed look into the MQTT topic design giving additional context which the reference page does not provide. It is assumed that you are familiar with the [MQTT Topic Schema](../../references/mqtt-api.md) page, if not, then please read it first before continuing.
+
+## Main concepts
+
+This sections provides some additional information regarding the MQTT topic design which is not explicitly mentioned in the [reference section](../../references/mqtt-api.md). It is intended to give contributors deeper understanding as to better know how to use and extend it in the future.
+
+### Identifier - fixed 4 segment group
+
+The **identifier** plays an important role in the topic design as it signifies which identity the command and telemetry data is associated with.
```

Comment:
A identifier is supposed to be the identifier of something ;-). 

Here it can be identifiers of:
- thin-edge entity/component
- source/target of MQTT messages

Reply:
The identifier identifies the identity (or is this too confusing for the user?)

Wiki's definition seems to reflect my usage (unless I'm not seeing something):

> An identifier is a name that identifies (that is, labels the identity of) either a unique object or a unique class of objects, where the "object" or class may be an idea, physical [countable](https://en.wikipedia.org/wiki/Countable) object (or class thereof), or physical [noncountable](https://en.wikipedia.org/wiki/Mass_noun) substance (or class thereof).

What about the following instead?
---

The **identifier** is used to group command and telemetry data to a given entity. The exact meaning given to the **identifier** is up to you. For example, it can be used to represent:

* thin-edge entity/component
* source/target of MQTT messages

Reply:
> The identifier identifies the identity (or is this too confusing for the user?)

Not sure this would help a user ;-) but we don't have to explain it.

> Wiki's definition seems to reflect my usage (unless I'm not seeing something):
> 
> > An identifier is a name that identifies (that is, labels the identity of) either a unique object or a unique class of objects, where the "object" or class may be an idea, physical [countable](https://en.wikipedia.org/wiki/Countable) object (or class thereof), or physical [noncountable](https://en.wikipedia.org/wiki/Mass_noun) substance (or class thereof).

Exactly! The point I'm missing in the current wording is the *object*. These are identifiers of *what*?

> What about the following instead?
> 
> The **identifier** is used to group command and telemetry data to a given entity. The exact meaning given to the **identifier** is up to you. For example, it can be used to represent:
> 
>     * thin-edge entity/component
> 
>     * source/target of MQTT messages

Even simpler:

The identifier is used to group command and telemetry data to a given entity: a device, a service or any source or target of MQTT messages.

Reply:
@didier-wenzek thanks, I used your sentence.